scripts/build.js

```diff
@@ -68,7 +68,7 @@ async function main() {
 
   // detect unexpected asset emissions from core build
   function checkUnknownAssets (buildName, assets) {
-    assets = assets.filter(name => !name.endsWith('.cache') && !name.endsWith('.cache.js') && name !== 'LICENSES.txt');
+    assets = assets.filter(name => !name.endsWith('.cache') && !name.endsWith('.cache.js') && !name.endsWith('LICENSES.txt'));
```

Comment:
Why change this?

Reply:
It ended up emitting a new LICENSES.txt asset, possibly from the webpack build itself.